go.mod

```diff
@@ -0,0 +1,9 @@
+module mattermost-developer-documentation
+
+go 1.13
+
+require (
+	github.com/mattermost/mattermost-server/v5 v5.18.0-rc.test
```

Comment:
:100:

Reply:
Didn't catch this at first, but that tag don't exist anymore, so we should point the server dep to a specific commit instead.

Reply:
Fixed. The pseudo-version selected seems "wrong", but the commit hash results in the desired behaviour. I suspect this will get fixed once v5.18.0 actually gets released.